solutions/devsprint-haldny-santos-1/app/build.gradle

```diff
@@ -60,9 +60,10 @@ android {
 }
 
 dependencies {
-    def lifecycle_version = "2.4.1"
+    def lifecycle_version = "2.5.1"
     def retrofit_version = "2.9.0"
     def moshi_version = "2.9.0"
+    def picasso_version = "2.8"
```

Comment:
Por quê você escolheu o Picasso? 

Você avaliou essas outras soluções?
https://medium.com/@manuaravindpta/glide-picasso-and-fresco-library-support-a099a978d44a

Reply:
Não avaliei outras soluções, escolhi Picasso por ter utilizado e entendi que iriamos discutir na call. Vou fazer a leitura indicada.